### ISSUE TITLE
Suggestion: Self-Rift opens Ender Inventory

### ISSUE THREAD

**Issue by Morinor** [Azanor/thaumcraft-suggestions#462]
HAving a way to access your enderchest without having to place it down is rly useful, and it would add a 
use to a otherwise pointless focus combination.

**Comment by mfk1016** [comment]
I like this idea, but I also think that a focus like this should be at least mid-tier. An ender pearl should still be the necessity for creating devices which allow access to the ender storage.

**Comment by Morinor** [comment]
True, it would be a little too cheap... maybe if the vis cost was much higher, it could be moe balanced?
Or the complexity for this particular combination would be much higher than it is now

**Comment by mfk1016** [comment]
Having the complexity on 20 would be enough to require the advanced focus (which is the mid tier one requiring an ender pearl to create)

**Comment by AmyMialee** [comment]
I think it would be nice to be a separate inventory, instead of just ender chest. Kinda like a second enderchest, so you have more reason to get it and progress in the mod.

**Comment by devtn2001** [comment]
Nice
